### PR TITLE
Json field check

### PR DESCRIPTION
**Pull request by cs-yash-patodkar**
Json field check in edit controller

### CODE REVIEW

File: widget/edit.html [hunk 1]
```diff
@@ -38,35 +38,41 @@ <h3 class="modal-title col-md-9">Edit Funnel Chart</h3> <button type="button" cl
...
-                        <span class="text-danger">*</span>
-                    </label>
-                    <div data-cs-conditional data-ng-if="fieldsArray.length > 0" data-fields="$parent.fields"
-                        data-reset-field="$parent.fields" data-mode="'queryFilters'"
-                        data-ng-model="$parent.config.query" data-parent-form="editFunnelWidgetForm"
-                        data-enable-expression="(page==='dashboard' || page==='reporting')" data-show-uuid="true"
-                        data-form-name="'editFunnelWidgetForm'">
+                <div class="form-group" data-ng-if="objectFields.length > 0">
+                    <div class="margin-top-md margin-bottom-md">
+                        <label class="control-label">Filter Record Which Contains The Data
```
Comment (cs-suyog-jinturkar) [inline widget/edit.html]: what about "Filter record which contains the JSON data"?

Comment (cs-yash-patodkar) [reply widget/edit.html]: Done

File: widget/edit.html [hunk 2]
```diff
@@ -75,18 +81,20 @@ <h6 style="margin-top: 0px;">Layer {{$index +1}}</h6>
                     <!-- Show value field only when funnelModuleType value is 1-->
                     <div data-ng-if="config.funnelModuleType == 1">
 
-                        <div data-ng-class="{ 'has-error': editFunnelWidgetForm['customModuleValue-'+$index].$invalid && editFunnelWidgetForm['customModuleValue-'+$index].$touched }">
+                        <div
```
Comment (cs-suyog-jinturkar) [inline widget/edit.html]: Why this change?

File: widget/edit.controller.js [hunk 3]
```diff
@@ -58,12 +57,12 @@
...
       $scope.fields = [];
       $scope.fieldsArray = [];
-      $scope.pickListFields = [];
+      $scope.objectFields = [];
       var entity = new Entity($scope.config.customModule);
       entity.loadFields().then(function () {
         for (var key in entity.fields) {
-          if (entity.fields[key].type === "picklist") {
-            $scope.pickListFields.push(entity.fields[key]);
+          if (entity.fields[key].type === "object") {
```
Comment (cs-suyog-jinturkar) [inline widget/edit.controller.js:65]: Please add some comments

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added some review comments

File: widget/edit.html (hunk 2 above)
Comment (cs-yash-patodkar) [reply widget/edit.html]: Space issue

File: widget/edit.controller.js (hunk 3 above)
Comment (cs-yash-patodkar) [reply widget/edit.controller.js:65]: Done

**Review by cs-suyog-jinturkar — APPROVED** [review]
